ansible/roles/netbox/defaults/main.yml

```diff
@@ -583,24 +579,31 @@ netbox__config_media_root: '{{ netbox__data + "/media" }}'
 # Absolute path where Netbox reports (Python modules) are stored.
 netbox__config_reports_root: '{{ netbox__data + "/reports" }}'
                                                                    # ]]]
+# .. envvar:: netbox__config_plugins [[[
+#
+# List of Netbox plugins.
+netbox__config_plugins: []
+                                                                   # ]]]
```

Comment:
You could add an empty line before `# ]]]` marker to have the variables better stand out in the defaults file.

Reply:
I guess I know. I just did not put so much effort into this because I wrote software that can lint/fix this.

> Never send a human to do a machine's job.

Ref: https://github.com/debops/debops/issues/1460